tests/adapters/switches/juniper_test.py

```diff
@@ -1184,35 +1207,46 @@ def test_get_interfaces(self):
             <vlans/>
         """))
 
-        if1, if2, if3, if4, if5 = self.switch.get_interfaces()
+        if1, if2, if3, if4, if5, if6, if7 = self.switch.get_interfaces()
```

Comment:
either have new interfaces or not but do the same in both tests, i prefer the less interfaces possible